collectors/okta/okta_collector.js

```diff
@@ -29,6 +30,12 @@ const tsPaths = [
 
 class OktaCollector extends PawsCollector {
 
+    constructor(context, {aimsCreds, pawsCreds}){
+        super(context,
+            {aimsCreds, pawsCreds},
+            [healthChecks.oktaTokenHealthCheck], []);
```

Comment:
I wonder if we can bind this function to `this` right here and avoid any changes in `al-aws-collector-js`. Also making `oktaTokenHealthCheck ` as a method of `OktaCollector` class

Reply:
I don't really see the value of making it a class member. But that is more a personal style view than one with a particular technical objection.

It would be simpler to bind here. But I can see the benefit of making it a general solution in `al-aws-collector-js`. we will likely have more health checks that need to access the actual class for one resason or another, and i don't think we're going to have health checks that refer to the actual function instance themselves.

But I can see the value of not touching al-aws-collector-js this close to GA. So I'm cool with binding here.